src/framework/piral-core/src/types/state.ts

```diff
@@ -36,15 +36,24 @@ declare module './components' {
 
 export type WrappedComponent<TProps> = ComponentType<Without<TProps, keyof BaseComponentProps>>;
 
+/**
+ * The base type of registration how the pilet is registered in the global state context.
```

Comment:
Not how the pilet is registered, but rather how components from a pilet are registered.

"The base type for pilet component registration in the global state context."

Reply:
adapted